services/management/commands/turku_service_import/accessibility.py

```diff
@@ -0,0 +1,148 @@
+from munigeo.importer.sync import ModelSyncher
+
+from services.management.commands.turku_service_import.utils import get_ar_resource, get_ar_servicepoint_resource, \
+    get_ar_servicepoint_accessibility_resource
+from services.models import Unit, AccessibilityVariable, UnitAccessibilityProperty
+
+VERBOSITY = False
+LOGGER = None
+
+
+class AccessibilityImporter:
+    def __init__(self, verbosity=True, logger=None):
+        self.verbosity = verbosity
+        self.logger = logger
+        self.__accessibility_variable_cache = {}
+
+    def import_accessibility(self):
+        self._import_accessibility_variables()
+        self._update_unit_accessibility_info()
+        self._import_unit_accessibility_properties()
+
+    @property
+    def _accessibility_variables(self):
+        if not self.__accessibility_variable_cache:
+            self.__accessibility_variable_cache = {variable.id: variable for variable in
+                                                   AccessibilityVariable.objects.all()}
+
+        return self.__accessibility_variable_cache
+
+    def _import_accessibility_variables(self):
+        num_of_imports = 0
+
+        variables = get_ar_resource('accessibility/variables')
+        for variable in variables:
+            variable_name = variable.get('variableName')
+            variable_id = variable.get('variableId')
+            if not (variable_name and variable_id):
+                continue
+            accessibility_variable, created = AccessibilityVariable.objects.get_or_create(
+                id=variable_id,
+                name=variable_name,
+            )
```

Comment:
I don't know much about the data in question, but might be still better to do
```
AccessibilityVariable.objects.update_or_create(
    id=variable_id,
    defaults={'name': variable_name}
)
```

so it won't break if a name happens to change?

Reply:
You a correct, this should have been made with `defaults`